torch/nn/modules/loss.py

```diff
@@ -1235,24 +1240,45 @@ class CTCLoss(_Loss):
 
     Inputs:
         log_probs: Tensor of size :math:`(T, N, C)` where `C = number of characters in alphabet including blank`,
-            `T = input length`, and `N = batch size`.
+        `T = input length`, and `N = batch size`.
             The logarithmized probabilities of the outputs
             (e.g. obtained with :func:`torch.nn.functional.log_softmax`).
-        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`.
-            Targets (cannot be blank). In the second form, the targets are assumed to be concatenated.
+        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`, where `N = batch size`, and
+        `S = maximum target length`.
```

Comment:
What's the effect of changing the indent here?

Reply:
Did you mean on line 1238?

The indent means that the remaining words in that sentence will belong to the header block when rendered. I noticed that some of the other docs don't do this, so I have actually changed it again to something else.